css/style.css

```diff
@@ -0,0 +1,453 @@
+/* Defaults */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color1: #f4f5f6;
+  --color2: #c7c7c7;
+  --color3: black;
+  --text-header: rgba(3, 3, 3, 0.4);
+  --main-color: #f72210;
+}
+
+body {
+  background-color: var(--color1);
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.border-temp {
+  border: 1.5px dashed blue;
+}
+
+.border-temp-1 {
+  border: 2px dashed rgb(12, 1, 1);
+}
+
+.border-temp-2 {
+  border: 3px dashed yellowgreen;
+}
+
+.hover-none:hover {
+  text-decoration: none;
+  color: white;
+}
+
+.hover-link-cards:hover {
+  text-decoration-color: #f72210;
+}
+
+.width-custom {
+  max-width: 1310px;
+}
+
+.news-catagory {
+  color: var(--text-header);
+  font-size: 16px;
+  font-weight: 600;
+  margin: 0.7rem 0;
+}
+
+.card {
+  border: unset;
+}
+
+.main-bg {
+  background-color: var(--main-color);
+}
+
+.white-bg {
+  background-color: #fff;
+}
+
+.main-color {
+  color: var(--main-color);
+}
+
+/* Fixed navbar starts here */
+
+.fixednav {
+  position: fixed;
+  top: 0;
+  z-index: 100;
+  height: 50px;
+}
+
+.avatar img {
+  width: 20px;
+}
+
+.fixednav-btn-login {
+  color: white;
+  font-size: large;
+}
+
+.fixednav-btn-subsc {
+  background-color: #ffe400;
+  font-size: 18px;
+  border-radius: 1rem;
+  height: 30px;
+}
+
+.fixednav-btn-subsc > p {
+  font-size: 11px;
+}
+
+.fixednav-btn-toggle {
+  width: 30px;
+  height: 100%;
+}
+
+.fixednav-btn-toggle > div {
+  width: 100%;
+  height: 4px;
+  background-color: rgb(199, 194, 194);
+}
+
+/* main Header starts here */
+
+.header-top {
+  height: 95px;
+}
+
+.weather > img {
+  width: 24px;
+  height: 24px;
+}
+
+.weather p {
+  font-size: 16px;
+}
+
+.header-top_logo > img {
+  height: 95%;
+  width: 400px;
+  margin: 0 0 0 8%;
+}
+
+.header-middle {
+  height: 53px;
+}
+
+.header-middle::before {
+  position: absolute;
+  top: 0;
+  right: 0;
+  content: "";
+  display: block;
+  width: 70px;
+  background-color: #f72210;
+  border-left: 12px solid white;
+  border-top: 13px solid #f72210;
+}
+
+.header-middle_links {
+  width: 88%;
+}
+
+.header-middle_links li {
+  border-right: 1px solid rgb(219, 209, 209);
+}
+
+.header-middle_links a {
+  padding: 0 calc(1.7vw - 10px);
+  color: black;
+  font-weight: 700;
+  font-size: 19px;
+  white-space: nowrap;
+}
+
+.header-middle_search > p {
+  font-size: 19px;
+}
+
+.header-middle_search > img {
+  height: 25px;
+  width: 25px;
+}
+
+.header-bottom {
+  height: 43px;
+  border-top: 2px #f4f5f6 solid;
+  border-bottom: 2px #f4f5f6 solid;
+}
+
+.header-bottom a {
+  margin: 0 1rem;
+  font-weight: 700;
+}
+
+.header-middle_links a:hover {
+  color: #f72210;
```

Comment:
- [ ] **[MANDATORY]** For good CSS practices ,use your --main-color variable here as well